conf/messages.cy

```diff
@@ -373,7 +376,7 @@ pla.exitSurvey.question2                                = <p class="bold-small">
 pla.exitSurvey.question3                                = <p class="bold-small">Pa mor debygol ydych o argymell eich cyfrif treth i''ch ffrindiau a''ch teulu?</p>
```

Comment:
Should we be adding styles etc to the message files - may need some investigation on subsequent story

Reply:
We definitely shouldn't be

Reply:
Sometimes it can't be avoided with the way a service has been coded, but we should definitely remove if we can